library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -354,23 +379,35 @@ interface ZoomApi {
         const val TRANSFORMATION_NONE = 2
 
         /**
-         * Constant for [ZoomApi.setSmallerPolicy].
-         * When smaller than its container, the content will be centered.
+         * Constant for [ZoomApi.setTransformation] gravity.
+         * This currently means that the gravity will be inferred from the alignment or
```

Comment:
currently?
This implies that this will change in the short term future. Will it?